bintray/services/upload.go

```diff
@@ -240,6 +241,16 @@ func getSingleFileToUpload(rootPath, targetPath string, flat bool) clientutils.A
 	return clientutils.Artifact{LocalPath: rootPathOrig, TargetPath: uploadPath}
 }
 
+func (up *UploadParams) GetPatternType() clientutils.PatternType {
```

Comment:
Notice that thw code of this method is duplicated. See the same method in ```ArtifactoryCommonParams```.
I suggest moving this code to ```github.com/jfrog/jfrog-client-go/utils``` and reusing it.